node/config.go

```diff
@@ -207,7 +208,7 @@ func NewConfig(ctx *cli.Context) (*Config, error) {
 		EnableNodeApi:                  ctx.GlobalBool(flags.EnableNodeApiFlag.Name),
 		NodeApiPort:                    ctx.GlobalString(flags.NodeApiPortFlag.Name),
 		EnableMetrics:                  ctx.GlobalBool(flags.EnableMetricsFlag.Name),
-		MetricsPort:                    ctx.GlobalString(flags.MetricsPortFlag.Name),
+		MetricsPort:                    ctx.GlobalInt(flags.MetricsPortFlag.Name),
```

Comment:
Is this backward compatible? i.e. if node doesn't update the env var from string to int, does it continue to work?

Reply:
Bash variables are untyped. From bash's perspective, everything is just a string (regardless of quotation mark usage). Changing the flag from a `GlobalString` to a `GlobalInt` just causes golang to parse the data into an int when it reads it.

https://tldp.org/LDP/abs/html/untyped.html